nsxt/resource_nsxt_policy_lb_https_monitor_profile_test.go

```diff
@@ -0,0 +1,258 @@
+/* Copyright © 2020 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyLBHttpsMonitorProfileCreateAttributes = map[string]string{
+	"display_name":            getAccTestResourceName(),
+	"description":             "terraform created",
+	"request_body":            "test-create",
+	"request_method":          "POST",
+	"request_url":             "test-create",
+	"request_version":         "HTTP_VERSION_1_1",
+	"response_body":           "test-create",
+	"response_status_codes":   "[200, 304]",
+	"fall_count":              "2",
+	"interval":                "2",
+	"monitor_port":            "2",
+	"rise_count":              "2",
+	"timeout":                 "2",
+	"header_name":             "test-create",
+	"header_value":            "test-create",
+	"certificate_chain_depth": "2",
+	"server_auth":             "test-create",
+}
+
+var accTestPolicyLBHttpsMonitorProfileUpdateAttributes = map[string]string{
+	"display_name":            getAccTestResourceName(),
+	"description":             "terraform updated",
+	"request_body":            "test-update",
+	"request_method":          "OPTIONS",
+	"request_url":             "test-update",
+	"request_version":         "HTTP_VERSION_1_0",
+	"response_body":           "test-update",
+	"response_status_codes":   "[200, 304]",
+	"fall_count":              "5",
+	"interval":                "5",
+	"monitor_port":            "5",
+	"rise_count":              "5",
+	"timeout":                 "5",
+	"header_name":             "test-update",
+	"header_value":            "test-update",
+	"certificate_chain_depth": "5",
+	"server_auth":             "test-update",
+}
+
+func TestAccResourceNsxtPolicyLBHttpsMonitorProfile_basic(t *testing.T) {
+	testResourceName := "nsxt_policy_lb_https_monitor_profile.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
```

Comment:
I don't think there is support for global manager, so we should add `testAccOnlyLocalManager(t)` here